src/main/webapp/index.html

```diff
@@ -38,6 +38,17 @@
           </div>
         </div>
       </div>
+    <div name="results-overlay"> 
+      <div class="modal" id="results-popup">
+        <div class="modal-header"> 
+          <div class="modal-title">Results</div>
+          <button data-modal-close-button class="exit-button">&times;</button>
```

Comment:
does data-modal-close-button have to equal anything? This is new to me, would love to learn.

Reply:
Yes, although this is a vacuous attribute, it makes it easier to grab the element by a document scan for attributes.